include/gatekeeper_gk.h

```diff
@@ -122,6 +140,13 @@ struct gk_fib {
 			 */
 			struct ether_cache eth_cache;
 		} grantor;
+
+		/*
+		 * When the action is GK_FWD_BACK_NET or GK_FWD_FRONT_NET,
+		 * we need the next fib entry for the gateway of the
+		 * destination.
+		 */
+		struct gk_fib *next_fib;
```

Comment:
A problem to think how to solve: an LPM prefix maps to a FIB entry whose action is `GK_FWD_NEIGHBOR_*_NET`, and when a packet hits that entry, there's no one subscribing with LLS the neighbor (i.e. the destination address of the packet). Who's going to add that neighbor in the hash table of neighbors since that table is global and many GK blocks could have the same need at the same time? Moreover, there's no other entry referring that new neighbor as is the case for grantor entries and gateway entries (as mentioned in the previous comment), so when/who should that neighbor be released?